webui/react/src/components/kit/Dropdown.tsx

```diff
@@ -66,12 +67,15 @@ const Dropdown: React.FC<PropsWithChildren<Props>> = ({
   }, [menu, onClick]);
 
   const renderedChildren = useMemo(() => {
-    return (
-      <div>
-        {/* wrap in div to prevent antd from overwriting classes for child button element */}
-        {children}
-      </div>
-    );
+    if ((isValidElement(children) && children?.type === 'button') || isForwardRef(children)) {
```

Comment:
Oof, is there an issue against antD for the fact that they're overwriting classes instead of adding to them?

Reply:
not that I could find. actually, taking a quick look at the source code, looks like they're appending. so I'm not really sure why I'm seeing this behavior.

Reply:
I would encourage you to figure out the underlying cause here.